libraries/ESP8266HTTPUpdateServer/examples/SecureHTTPSUpdater/SecureHTTPSUpdater.ino

```diff
@@ -0,0 +1,144 @@
+/*
+  To upload through terminal you can use: curl -u admin:admin -F "image=@firmware.bin" esp8266-webupdate.local/firmware
+*/
```

Comment:
Please add a public domain/CC0 licence header to this file.
E.g.

https://github.com/esp8266/Arduino/blob/master/libraries/ESP8266WiFi/examples/WiFiEvents/WiFiEvents.ino#L12-L13

Reply:
Done.  New header on the 3 examples includes public domain release, info on using and making SSL certs, browser interactions, etc.